pubsub/natspubsub/example_test.go

```diff
@@ -52,7 +52,14 @@ func ExampleCreateSubscription() {
 	}
 	defer nc.Close()
 
-	sub := natspubsub.CreateSubscription(nc, "go-cloud.example.send")
+	ackFunc := func() {
+		// This function will be called when the application calls "Ack" on a
+		// received message.
+		// Since Ack is a meaningless no-op for NATS, you can provide an empty
+		// function to do nothing, or panic/log a warning if your application
```

Comment:
This comment is fine. But FTR, I'm more concerned about applications that are build for at-*least*-once semantics and *do* call ack, but inadvertently get linked with with an at-most-once provider.

Reply:
Ack.

Reply:
:-)